web/src/main/react-dashboard/src/components/TestCaseView.js

```diff
@@ -83,7 +83,10 @@ class TestCaseView extends Component {
                                 <TableRowColumn>
                                     <h4>{data.name}</h4></TableRowColumn>
                                 <TableRowColumn><SingleRecord value={data.success} /></TableRowColumn>
-                                <TableRowColumn style={{ color: 'red' }}>
+                                <TableRowColumn style={{
+                                    color: 'red', whiteSpace: 'normal',
```

Comment:
Shall we have `, whiteSpace: 'normal',` in a new line so that the style is consistent and more readable